### DISCUSSION TITLE
Importation Visual Studio

### DISCUSSION THREAD

**Discussion by PhilippeLRETQC** [MTESSDev/FRW#17] [Annonces]
Bonjour,

Le fichier https://github.com/MTESSDev/FRW/blob/main/Swagger/FRW.API.yml n'est pas importable directement dans Visual Studio à cause de l'extension. Il faudrait changer de yml à yaml. En général, c'est aussi partagé sous forme de json.

Bonne journée.

**Comment by anisite** [comment]
Bonjour, j'ai effectué le changement mais normalement les deux extensions sont supportées un peu partout ? 

Il est possible aussi de convertir en json ici https://editor.swagger.io/ ou encore d'explorer le YAML plus facilement, il y a certaines incompatibilitées par contre avec le `additionnals attributes` qui donne des résultats douteux.

Dans l'idéal on devrait se faire un projet sample partageable qui implémente les différents appels, si jamais vous en avez un on pourrait l'ajouter à ce dépot GIT
